gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/QueryServiceRequestAndResponseUtils.java

```diff
@@ -162,29 +175,31 @@ public static Filter createQsDefaultRequestFilter(String timestampColumnName,
                                                     long startTime,
                                                     long endTime) {
     return Filter.newBuilder()
-        .addChildFilter(
-            createQsFilter(
-                createQsColumnExpression(timestampColumnName),
-                Operator.GE,
-                createQsStringLiteralExpression(Long.toString(startTime))
-            )
-        )
-        .addChildFilter(
-            createQsFilter(
-                createQsColumnExpression(timestampColumnName),
-                Operator.LT,
-                createQsStringLiteralExpression(Long.toString(endTime))
-            )
-        )
+        .setOperator(Operator.AND)
         .addChildFilter(
             Filter.newBuilder()
+                .setOperator(Operator.AND)
                 .addChildFilter(
                     createQsFilter(
-                        createQsColumnExpression(entityIdColumnName),
-                        Operator.NEQ,
-                        createQsStringLiteralExpression("null")
+                        createQsColumnExpression(timestampColumnName),
+                        Operator.GE,
+                        createQsLongLiteralExpression(startTime)
                     )
                 )
-        ).build();
+                .addChildFilter(
+                    createQsFilter(
+                        createQsColumnExpression(timestampColumnName),
+                        Operator.LT,
+                        createQsLongLiteralExpression(endTime)
+                    )
+                ))
+        .addChildFilter(
```

Comment:
Is this added twice?

Reply:
No, i see it only once.

Reply:
Might be unrelated to this change but this exists in QueryServiceEntityFetcher.constructQueryServiceFilter() as well. Which is also coincidentally the method that adds time range to the query service request.

Reply:
Correct. That will now be removed since time range will not be special case anymore with this change.

Reply:
In another PR?

Reply:
This is already fixed in this PR.